pixi.toml

```
@@ -0,0 +1,81 @@
+[project]
+name = "metalearners"
+channels = ["conda-forge", "nodefaults"]
+platforms = ["linux-64", "osx-arm64", "osx-64", "win-64"]
+
+[tasks]
+postinstall = "pip install --no-build-isolation --no-deps --disable-pip-version-check -e ."
+coverage = "pytest tests --cov=metalearners --cov-report=xml --cov-report term-missing --color=yes"
+
+[dependencies]
+pre-commit = "*"
+git_root = "*"
+pip = "*"
+python = ">=3.10"
+setuptools-scm = "*"
+setuptools = ">=61"
+ipykernel = "*"
+ipywidgets = "*"
+lime = "*"
+optuna = "*"
+nbconvert = "*"
+mypy = "1.9.0.*"
```

```suggestion
mypy = "*"
```

mypy is already being locked in `pixi.lock`, `*` should be fine